doc:wifi: Add a section about nRF7002 EK

richabp opened the pull request:
Added a section about nRF7002 EK in the Getting started guide. SHEL-1826

NordicBuilder commented:
You can find the documentation preview for this PR at this <a href='http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-12021
'>link</a>. It will be updated about 10 minutes after the documentation build succeeds.

*Note: This comment is automatically posted by the Documentation Publishing GitHub Action.*

ioannisg commented on doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst:
```diff
@@ -1,16 +1,25 @@
-   :depth: 2
+   :depth: 4
+
+This section gets you started with your nRF70 Series :term:`Development Kit (DK)` using the |NCS|.
```
I think this intro text needs to be more generic. It covers all evaluation boards - not just the DK. 

This exact text could potentially be reused in the DK section.

ioannisg commented on doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst:
```diff
@@ -1,16 +1,25 @@
+   :depth: 4
+
+This section gets you started with your nRF70 Series :term:`Development Kit (DK)` using the |NCS|.
+It tells you how to perform a quick test of your DK.
```
Not applicable at this level.

ioannisg commented on doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst:
```diff
@@ -1,16 +1,25 @@
-********
+--------
 
 The nRF7002 is an IEEE 802.11ax (Wi-Fi 6) compliant solution that implements the Wi-Fi physical layer and MAC layer protocols.
```
Since we are reviewing this once more. Guess it is nice to have MAC abbreviation also explained (as for physical layer). So medium access control, it should be

ioannisg commented on doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst:
```diff
@@ -245,3 +254,118 @@ The following samples allow you to test if the buttons (or switches) and LEDs on
+
+The nRF7002 EK (PCA63556) is designed to provide Wi-Fi connectivity and Wi-Fi (SSID) scanning capabilities via the nRF7002 companion IC to a compatible host development board.
+
+The nRF7002 EK features an Arduino shield form factor and interface connector that allows it to be used with Arduino compatible boards, such as the nRF52840 DK, nRF5340 DK, or nRF9160 DK.
```
Do not we need active links to these DK product pages (or corresponding NCS doc pages)??

richabp replied:
Added links to the DK product pages.

ioannisg commented on doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst:
```diff
@@ -245,3 +254,118 @@ The following samples allow you to test if the buttons (or switches) and LEDs on
+
+To add support for the nRF7002 EK on an application running on a compatible host development board, the ``SHIELD`` setting must be specified.
+
+The nRF7002 EK features the nRF7002 companion IC.
```
I am thinking now, this sentence could alternatively be moved in the overview section, above.
Also: do we need active links to the 7000 and 7001 product pages? Or, the common nrf70 series landing page?

richabp replied:
Added the sentence above the "Overview section". 

Added the link to the common nrf70 series landing page in the Reference section.

ioannisg commented on doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst:
```diff
@@ -245,3 +254,118 @@ The following samples allow you to test if the buttons (or switches) and LEDs on
+
+Alternatively, add the shield in the project's :file:`CMakeLists.txt` file, specifying the below settings, depending on which IC is to be used:
+
+.. code-block:: console
```
would it be better looking if we merged these code blocks in one, with multiple lines?

richabp replied:
These commands are referring to individual ICs, hence separate code blocks.
Generally, a series of following commands are listed in the same code block.
But, if separate code blocks are not serving the purpose, I will change.

ioannisg commented on doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst:
```diff
@@ -245,3 +254,118 @@ The following samples allow you to test if the buttons (or switches) and LEDs on
+.. code-block:: console
+
+   west build -b nrf5340dk_nrf5340_cpuapp -- -DSHIELD=nrf7002ek
+
```
maybe just add a sentence explaining that for emulated board options similar commands are needed, (without necessarily listing the commands)

ioannisg commented on doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst:
```diff
@@ -245,3 +254,118 @@ The following samples allow you to test if the buttons (or switches) and LEDs on
+----------
+
+* `nRF70 Series product page`_
+* `nRF70 Series`_
```
Can we include "infocenter" here? Somehow in needs to differ from the description above since one is pointing to nordicsemi.com and the latter to the infocenter...

richabp replied:
Added "hardware documentation".

ioannisg commented on doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst:
```diff
@@ -245,3 +254,118 @@ The following samples allow you to test if the buttons (or switches) and LEDs on
+
+* `nRF70 Series product page`_
+* `nRF70 Series`_
+* `Product specification for nRF70 Series devices`_
```
This directly points to the 7002 PS, not the nRF70 Series. So it is not ideal. We also have 7001 PS available. Can rename to 7002 and also add a reference link to 7001 PS?

NordicBuilder commented:
## Test specification

### [CI/Jenkins/NRF](https://jenkins-ncs.nordicsemi.no/job/latest/job/sdk-nrf/job/PR-12021/3/)
- Skipped

### [CI/Jenkins/integration](https://jenkins-ncs.nordicsemi.no/job/latest/job/sdk-nrf/job/PR-12021/3/)
- Skipped



*Note: This message is automatically posted and updated by the CI*

ioannisg reviewed (commented):
This looks fine to me (one comment left regarding the sentence to be moved).
But will need some review help from the Wi-Fi team

CC @krish2718 @udaynordic @sachinthegreen

bama-nordic commented on doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst:
```diff
@@ -245,3 +253,121 @@ The following samples allow you to test if the buttons (or switches) and LEDs on
+
+   west build -b nrf5340dk_nrf5340_cpuapp -- -DSHIELD=nrf7002ek
+
+To build for the nRF7002 EK and the nRF7001 or nRF7000 ICs, you can use similar commands.
```
Maybe replace "use similar commands" to "use the corresponding shield name in the above command"

bama-nordic reviewed (approved):
Have a minor comment, LGTM

richabp commented:
> This looks fine to me (one comment left regarding the sentence to be moved). But will need some review help from the Wi-Fi team
> 
> CC @krish2718 @udaynordic @sachinthegreen

Addressed the comment.
